res/database/Default/database_ads.xml

```diff
@@ -2099,11 +2101,11 @@ Available in any well-stocked drugstore.</en>
 		<ad id="speedminister-ad-bafm-01" creator="8605" created_by="SpeedMinister">
 			<title>
 				<de>Bundesdeutsche Anstalt für Meditation</de>
-				<en>Federal Institution for Meditation</en>
+				<en>Federal Institute for Meditation</en>
 			</title>
 			<description>
 				<de>Leitfaden zur aufrichtigen inneren Reinigung, Tipps und Tricks, Best Practices und vieles mehr. Infos bei uns! LAM - VAM - RAM - YAM - HAM - KSHAM - OM.</de>
-				<en>Guide to sincere inner cleansing, tips and tricks, best practices and much more. Info from us! LAM - VAM - RAM - YAM - HAM - KSHAM - OM.</en>
+				<en>Guide to sincere inner cleansing, tips and tricks, best practices and much more. Info at our site! LAM - VAM - RAM - YAM - HAM - KSHAM - OM.</en>
```

Comment:
intended meaning not quite clear: "Infos on-site" (only when visiting) or just; "contact us for infos"

Reply:
ack (where's SpeedMinister?)

Changing to 'Contact us for more info'.
Probably the intended meaning.

Reply:
think SpeedMinister went on to other stuff.